### PR TITLE
Input help labels functionality

### PR DESCRIPTION
**Pull request by AwesomeFlax**
image
image

image

image

### CODE REVIEW

**Review by doskachok — COMMENTED** [review]
Nice! Should we also add * to the DisplayName if it is a required field?

File: src/components/Input/index.tsx [hunk 1]
```diff
@@ -88,7 +88,12 @@ const Input =
 
         {(required && !value) && <RequiredIndicator reversedTheme={reversedTheme}>*</RequiredIndicator>}
 
-        {(!displayError && value && displayName) && <DisplayName reversedTheme={reversedTheme}> * {displayName} </DisplayName> }
+        {
+          (!displayError && value && displayName) && 
+          <DisplayName reversedTheme={reversedTheme}> 
+            {`${required ? '*' : ''} ${displayName}`.trim()}
```
Comment (doskachok) [inline src/components/Input/index.tsx:94]: Maybe do it like this, without trim?
```{`${required ? '* ' : ''}${displayName}`}```

Comment (AwesomeFlax) [reply src/components/Input/index.tsx:94]: nope, I want it to be
`* Display name`, not
`*Display name`

Comment (AwesomeFlax) [reply src/components/Input/index.tsx:94]: uh, you hid space in asterisk string. well, same shit, idk if it worth changing it

Comment (doskachok) [reply src/components/Input/index.tsx:94]: Yes,  there is a space after '* '

**Review by doskachok — APPROVED** [review]

**Pull request merged**